src/utils/filterPolicy.ts

```diff
@@ -0,0 +1,84 @@
+import { ConditionalOrderParams } from "@cowprotocol/cow-sdk";
+
+export enum FilterAction {
+  DROP = "DROP",
+  SKIP = "SKIP",
+  ACCEPT = "ACCEPT",
+}
+
+interface PolicyConfig {
+  owners: Map<string, FilterAction>;
+  handlers: Map<string, FilterAction>;
+}
+
+export interface FilterParams {
+  owner: string;
+  conditionalOrderParams: ConditionalOrderParams;
+}
+
+export interface FilterPolicyParams {
+  configBaseUrl: string;
+  // configAuthToken: string; // TODO: Implement authToken
+}
+export class FilterPolicy {
+  protected configUrl: string;
+  protected config: PolicyConfig | undefined;
+
+  constructor({ configBaseUrl }: FilterPolicyParams) {
+    this.configUrl = configBaseUrl;
+  }
+
+  /**
+   * Decide if a conditional order should be processed, ignored, or dropped base in some filtering rules
+   *
+   * @param filterParams params required for the pre-filtering, including the conditional order params, chainId and the owner contract
+   * @returns The action that should be performed with the conditional order
+   */
+  preFilter({ owner, conditionalOrderParams }: FilterParams): FilterAction {
+    if (!this.config) {
+      return FilterAction.ACCEPT;
+    }
+
+    const { owners, handlers } = this.config;
+
+    const action =
+      owners.get(owner) || handlers.get(conditionalOrderParams.handler);
```

Comment:
As a suggestion, are we sure we want to consider the individual owner case first? This may open up attacks if there's a public list with an owner explicitly whitelisted, in which case, one can create a bad handler and target that owner address with fake events being emitted.

Reply:
For now don't make sense, as the default is to ACCEPT. It makes sense if we allow to change the default to DROP or SKIP (by the way, good feature and easy to add)

However, if we change it in the future, we might not remember this, so changing it just in case.